src/renderer.js

```diff
@@ -4,16 +4,12 @@ const Bus = require("./bus")
 const Autodect = require("./autodetect")
 const Session = require("./session")
 const Macros = require("./macros")
-const CustomCSS = require("./storage/custom-css")
+//const CustomCSS = require("./storage/custom-css")
 const Theme = require("./storage/theme")
 const Settings = require("./settings")
 
 window.messages = window.messages || []
 
-CustomCSS.injectCSS().then(() =>
-  document.body.classList.remove("loading")
-)
```

Comment:
Could do in another branch... but if we're removing Custom CSS in favor of themes, we should probably just remove the files entirely. 

The `document.body.classList.remove("loading")` was a smart move though to stop that Flash of Wrong Theme thing, ideally we'd port that to themes as they flash now.

Reply:
Yeah, I was going to clean this up so the loading mechanism is ported to the theme logic